### PR TITLE
486 Set permissive umask.

### PR DESCRIPTION
**Pull request by pillarsdotnet**
Fixes: #486

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -17,6 +17,7 @@
 # @param provider What provider to use for installation of the packages, except gunicorn and Python itself.
 # @param use_epel to determine if the epel class is used.
 # @param manage_scl Whether to manage core SCL packages or not.
+# @param umask The default umask for invoked exec calls.
```
Comment (Dan33l) [inline manifests/init.pp]: Needs to update `REFERENCE.md` with following command :
`puppet strings generate --format markdown --out REFERENCE.md`

Comment (pillarsdotnet) [reply manifests/init.pp]: Done.

**Comment by Dan33l** [comment]
@pillarsdotnet please squash the 2 commits into 1

**Comment by pillarsdotnet** [comment]
As requested.

By the way; some projects squash commits when merging and frown on force-pushes because they obscure reviews and comments.

**Comment by Dan33l** [comment]
LGTM

**Pull request merged**